libusb/hid.c

```diff
@@ -1582,8 +1666,31 @@ int HID_API_EXPORT hid_send_feature_report(hid_device *dev, const unsigned char
 		(unsigned char *)data, length,
 		1000/*timeout millis*/);
 
-	if (res < 0)
+	if (res < 0) {
+		wchar_t *custom_string = NULL;
+
+		switch (res) {
+		case LIBUSB_ERROR_TIMEOUT:
+			custom_string = L"Transfer timed out";
+			break;
+		case LIBUSB_ERROR_PIPE:
+			custom_string = L"Control request not supported by device";
+			break;
+		case LIBUSB_ERROR_NO_DEVICE:
+			custom_string = L"Device has disconnected";
+			break;
+		case LIBUSB_ERROR_BUSY:
+			custom_string = L"Called from event handling context";
```

Comment:
that's not nesessary true

Reply:
My oppinion on this is that best to return `LIBUSB_ERROR_BUSY` as is, and leave the interpretation/digging documentation to the user of HIDAPI, and not try to hard-code things that might be changed in the future (e.g. updated documentation for libusb, etc.).